master/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml

```diff
@@ -78,6 +78,13 @@ rules:
       - list
       - update
       - watch
+  - apiGroups: [""]
```

Comment:
I think you can just add `- serviceaccounts` to the block at the top which includes namespaces.